packages/paste-core/components/chat-composer/__tests__/index.spec.tsx

```diff
@@ -26,15 +26,15 @@ const baseConfig = {
 
 describe('ChatComposer', () => {
   it('should render with placeholder text', () => {
-    render(<ChatComposer testid="my-composer" placeholder="Type here.." config={baseConfig} />);
+    render(<ChatComposer data-testid="my-composer" placeholder="Type here.." config={baseConfig} />);
     expect(screen.getByRole('textbox')).toBeDefined();
     expect(screen.getByText('Type here..')).toBeDefined();
   });
 
   it('should pass props to the content editable', async () => {
     render(
       <ChatComposer
-        testid="my-composer"
+        data-testid="my-composer"
```

Comment:
I have no idea how these tests were actually working. `testid` isn't a valid html prop, it should never have worked.